README.md

```diff
@@ -115,29 +115,38 @@ and ember-infinity will be set up to parse the total number of pages from a JSON
 }
 ```
 
-You may override `updateInfinityModel` to customize how the route's `model` should be updated with new objects.  Let's say we only want to show objects with `isPublished === true`:
+### Cursor-based pagination
 
-```js
-updateInfinityModel(newObjects) {
-  let infinityModel = this.get(this.get('_modelPath'));
-  
-  let content = newObjects.get('content');
-  let filtered = content.filter(obj => { return obj.get('isPublished'); });
-  
-  return infinityModel.pushObjects(filtered);
-}
-```
+If you are serving a continuously updating stream, it's helpful to keep track
+of your place in the list while paginating, to avoid duplicates. This is known
+as **cursor-based pagination** and is common in popular APIs like Twitter,
+Facebook, and Instagram. Instead of relying on `page_number` to paginate,
+you'll want to extract the `min_id` or `min_updated_at` from each page of
+results, so that you can fetch the next page without risking duplicates if new
+items are added to the top of the list by other users in between requests.
 
-You may also invoke this method directly to manually push new objects into the model:
+To do this, implement the `afterInfinityModel` hook as follows:
 
 ```js
-actions: {
-  pushHughsRecordsIntoInfinityModel() [
-    var updatedInfinityModel = this.updateInfinityModel(Ember.A([
-      { id: 1, name: "Hugh Francis Discography", isPublished: true }
-    ]));
-    console.log(updatedInfinityModel);
-  }
+_minId: undefined,
+_minUpdatedAt: undefined,
+_canLoadMore: true,
+
+model() {
+  return this.infinityModel("post", {}, {
+    min_id: '_minId',
+    min_updated_at: '_minUpdatedAt'
+  });
+},
+
+afterInfinityModel(posts) {
+  loadedAny = posts.get('length') > 0;
+  @set('_canLoadMore', loadedAny);
+
+  @set('_minId', posts.get('lastObject.id'));
+  @set('_minUpdatedAt', posts.get('lastObject.updated_at').toISOString());
+
+  return posts;
```

Comment:
can we please use `this` rather than `@`, want to be sure this isn't confusing for new devs